### PR TITLE
Add Cineplex

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: entries/c/cineplex.com.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "Cineplex": {
+    "domain": "cineplex.com",
+    "tfa": [
+      "call",
+      "sms"
+    ],
+    "documentation": "https://www.cineplex.com/policy/privacy-policy",
```
Comment (Carlgo11) [inline entries/c/cineplex.com.json]: Whilst the document you linked to does mention 2FA, it's not clear how 2FA is enabled and used. I recommend this page instead:
```suggestion
    "documentation": "https://cineplex.service-now.com/gsc?id=kb_article&sysparm_article=KB0011023",
```

File: img/c/cineplex.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....2 0-2.3.4-4.3 1.3-6.1.8-1.8 2.1-3.3 3.6-4.5 1.6-1.2 3.4-2.1 5.5-2.7 2.1-.6 4.4-.9 7-.9 2.1 0 4.9.2 6.7.5 1 .2 1.9.4 2.7.6 0 0 2.4.6 5.4 2.5l5.5-5.7zM310.3 57l41.3 42h11.5l-41.4-42z" class="st0"/><path d="m363.1 57-41.4 42h-11.4l41.4-42zM115 99.1V56.9h-8.1v30.5L77 56.9h-8.1v42.2H77V68.7l29.9 30.4z" class="st0"/><path d="M263.1 22.1c0-11.7-33.7-21.3-76.3-22.1l-.4 3.2c35.2.8 62.9 9.3 62.9 19.6 0 10.6-29 19.2-65.4 19.7l.3 1.6c43.8-.4 78.9-10.1 78.9-22" class="st1"/><path d="M232.5 24.7c0 7.9-21.8 14.4-49.2 14.8l.3 1.6c31.9-.4 57.3-8 57.3-17.3 0-9-24.1-16.4-54.9-17.2l-.4 3.2c26.3.9 46.9 7.2 46.9 14.9" class="st1"/><path d="M224.2 25.7c0-6.4-17-11.7-38.9-12.3l-.4 3.2c17.4.6 30.9 4.8 30.9 9.9 0 5.3-14.6 9.6-33.1 9.9l.3 1.6c23-.2 41.2-5.6 41.2-12.3M98.3 22.1C98.3 10.4 132 .8 174.7 0l.4 3.2c-35.2.8-62.9 9.3-62.9 19.6 0 10.6 29 19.2 65.4 19.7l-.3 1.6c-43.9-.4-79-10.1-79-22" class="st1"/><path d="M128.9 24.7c0 7.9 21.8 14.4 49.2 14.8l-.3 1.6c-31.9-.4-57.3-8-57.3-17.3 0-9 24.1-16.4 54.9-17.2l.4 3.2c-26.3.9-46.9 7.2-46.9 14.9" class="st1"/><path d="M137.2 25.7c0-6.4 17-11.7 38.9-12.3l.4 3.2c-17.4.6-30.9 4.8-30.9 9.9 0 5.3 14.6 9.6 33.1 9.9l-.3 1.6c-23-.2-41.2-5.6-41.2-12.3" class="st1"/></svg>
```
Comment (Carlgo11) [inline img/c/cineplex.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 53 14"><path id="a" fill="#ffd500" d="M7.04 7.56c0-2.83 7.58-5.15 17.27-5.41L24.05 0C10.59.28 0 3.29 0 6.95c0 3.75 11.13 6.82 25.06 6.96l.1-.92c-10.09-.14-18.12-2.52-18.12-5.43Zm18.34 3.32c-5.34-.1-9.58-1.03-9.58-2.56 0-1.47 3.92-2.37 8.98-2.54l-.2-1.62c-7.61.22-12.9 1.45-12.9 3.67 0 2.3 5.66 3.77 13.6 3.9z"/><use href="#a" transform="matrix(-1 0 0 1 53 0)"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Made Changes!

**Review by Carlgo11 — APPROVED** [review]
Looks good :wink: Thank you!

**Pull request merged**